examples/quickstart/tutorial/hadoop/docker/Dockerfile

```diff
@@ -1,3 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This comes from https://github.com/sequenceiq/hadoop-docker so the header here is probably not right. That is a third party work (https://www.apache.org/legal/src-headers.html#3party). The original file didn't have a license header, but it was distributed along with an Apache 2.0 LICENSE file, so I think it's clear that the original file is Apache licensed.

I don't see a clear policy about what to do here, but maybe a header like this will get the point across. It's the non-ASF version of the Apache 2.0 header with an extra sentence at the top.

```
# Based on the SequenceIQ hadoop-docker project hosted at
# https://github.com/sequenceiq/hadoop-docker, and modified at
# the Apache Software Foundation (ASF).
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
# 
#     http://www.apache.org/licenses/LICENSE-2.0
# 
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```